modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.js

```diff
@@ -12,36 +12,53 @@
  * details.
  */
 
-import {ClayButtonWithIcon} from '@clayui/button';
+import ClayButton from '@clayui/button';
 import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React, {useContext, useState} from 'react';
 
 import FrontendDataSetContext from '../../FrontendDataSetContext';
 import {triggerAction} from '../../utils/actionItems/index';
 
-function CreationMenu({primaryItems}) {
+function CreationMenu({inEmptyState, primaryItems}) {
 	const frontendDataSetContext = useContext(FrontendDataSetContext);
 
 	const {loadData} = frontendDataSetContext;
 
 	const [active, setActive] = useState(false);
 
+	const labelNew = Liferay.Language.get('new');
+	const buttonLabel = primaryItems[0].label ?? labelNew;
```

Comment:
If `primaryItems` is not set, or set to an empty array, this will break in an NPE. 

In existing code, the line can be like this because it is behind `primaryItems?.length > 0` and `primaryItems.length > 1`.